### ISSUE TITLE
Options missing in third-party loader

### ISSUE THREAD

**Issue by ogonkov** [webpack-contrib/expose-loader#114]
- Operating System: macos 10.15.6
- Node Version: v12.18.3
- NPM Version: 6.14.6
- webpack Version: 4.44.1
- expose-loader Version: 1.0.0

### Expected Behavior

Third-party loader (`fest-webpack-loader` in example) is receive options.

### Actual Behavior

When `expose-loader` is used options are just an empty object.

### Code

https://gist.github.com/ogonkov/0ca19648bebabf8ce2d3cfbb5a82b5e0

### How Do We Reproduce?

```
npm run build -- --mode=development index.js
```

`test.xml` is compiled without options to ES module. When `expose-loader` removed from rule, it compiled to CJS as expected. If insert `console.log` to `fest-webpack-loader` it's clear that it receive `{}` instead of loader options.

**Comment by jdnichollsc** [comment]
> > You probably want to put warning somewhere that other loader options is not respected by expose-loader.
> 
> It is not true, you need to use `getOptions` to get options from webpack configuration

Sorry folks, I'm getting this error: 
```
 ERROR  Failed to compile with 1 error                                               17:58:01

 error  in ./node_modules/phaser-ce/build/custom/p2.js

Syntax Error: TypeError: this.getOptions is not a function
```

Do you have any idea? Thanks for your help!

**Comment by alexander-akait** [comment]
You need webpack v5 to use latest version of this loader

**Comment by jdnichollsc** [comment]
@alexander-akait thanks Alexander, you're right! It was solved using expose-loader@1.0
